### PR TITLE
Dev can read function

### PR DESCRIPTION
**Pull request by autosarzs**
Test Pull Request

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Mozilla Public License Version 2.0
+Mozilla Public License Version 2.0
```
Comment (smsm2014) [inline LICENSE:1]: comment change this.

File: Software/bsw/gen/CanIf_Cbk.h [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  #error "The AR version of Std_Types.h does not match the expected version"
+#endif
+
+
+typedef uint32 PduIdType;
+
+void CanIf_TxConfirmation(PduIdType CanTxPduId);
+
+
+void CanIf_ControllerModeIndication( uint8 ControllerId, Can_ControllerStateType ControllerMode );
```
Comment (smsm2014) [inline Software/bsw/gen/CanIf_Cbk.h:31]: change name

**Review by smsm2014 — CHANGES_REQUESTED** [review]

File: Software/bsw/gen/CanIf_Cbk.h (hunk 2 above)
Comment (autosarzs) [reply Software/bsw/gen/CanIf_Cbk.h:31]: Done

File: LICENSE (hunk 1 above)
Comment (autosarzs) [reply LICENSE:1]: Dosn't matter it okay to be like this

**Pull request closed without merging**